resources/dicts/thoughts/alive/general.json

```
@@ -422,8 +428,10 @@
     {
         "id": "adventurous_gen_to_alive_kits",
         "thoughts": [
-            "Involves the kits in a game of make-believe about traveling through a strange land",
-            "Encourages the kits to explore as much as possible"
+            "Involves the kits in a game of make believe about traveling through a strange land",
```

```suggestion
            "Involves the kits in a game of make-believe about traveling through a strange land",
```
'make believe' is a phrase (eg i make believe that...) while 'make-believe' is a noun ('unicorns are make-believe').
sorry for the english lesson lol, i wasn't sure how to explain